_inc/social-logos.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Globally registers the 'social-logos' style and font.
+ *
+ * This ensures any theme or plugin using it is on the latest version of Social Logos, and helps to avoid conflicts.
+ */
+add_action( 'init', 'jetpack_register_social_logos', 1 );
+function jetpack_register_social_logos() {
+	if ( ! wp_style_is( 'social-logos', 'registered' ) ) {
+		wp_register_style( 'social-logos', plugins_url( 'social-logos/social-logos.css', __FILE__ ), false, '1' );
+	}
+}
```

Comment:
For consistency with the JavaScript coding guidelines, I usually try to leave an extra newline in PHP files as well. In this case, it looks like the issue is also on the WPCOM side, so not sure that we need to fix it in this PR. I'll let you be the judge.